lib/daru/maths/statistics/vector.rb

```diff
@@ -7,6 +7,8 @@ module Statistics
       module Vector
         def mean
           @data.mean
+          rescue ZeroDivisionError
```

Comment:
What is there on [line 87](https://github.com/v0dro/daru/blob/master/lib/daru/accessors/array_wrapper.rb#L87). Why should the conditional be removed?